nodejs/week2/warmup/index.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+const app = express();
+
+// GET /numbers/add?first=<number here>&second=<number here>. In response send sum (first + second).
+
+app.get("/numbers/add", (req, res) => {
```

Comment:
Looks good, but according to the task, you had to use `queries` and `parameters` for different operations. Please check the homework 😉